.github/workflows/ci_cd.yml

```diff
@@ -114,6 +113,12 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.10", "3.11", "3.12"]
+        server-version: ["latest"]
+        include:
+          - python-version: "3.12"
```

Comment:
What is the reason that 2024R2 and 2025R1 are tested only with Python 3.12?

Reply:
Just to limit the total number of tests somewhat. I'd expect changes which break on a different server version to be somewhat independent of those that break on a different Python version.

If you prefer, I can change it to test the full matrix.